src/__tests__/passwordReminderReducer.test.js

```diff
@@ -1,270 +0,0 @@
-// @flow
```

Comment:
Why did you remove this file?
Isn't it a working test?

Reply:
I didn't think reducer files were supposed to be in the tests?

Reply:
PasswordReminderModal.js 
is the only password reminder file in themed components